src/app/core/services/personal-cards.service.ts

```
@@ -53,9 +57,85 @@
+    });
+  }
+
+  mapPublicQueryParamsToPlatform(queryParams: {
+    btxn_status?: string;
+    ba_id?: string;
+    _search_document?: string;
+    amount?: string;
+    or?: string[];
+  }): PlatformPersonalCardQueryParams {
+    let q: string | undefined;
+    if (queryParams._search_document && queryParams._search_document.includes('fts.')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)